code/modules/admin/verbs/individual_logging.dm

```diff
@@ -5,7 +5,7 @@
 	var/ntype = text2num(type)
 
 	//Add client links
-	var/dat = ""
+	var/list/dat = list()
```

Comment:
Why make dat a list?

Reply:
I was told it would make other things more performant, due to how Byond handles text..

Reply:
list addition is much better performant than text one
this is a pattern not only here